pages/linux/lid.md

```diff
@@ -0,0 +1,12 @@
+# lid
+
+> Display a user's groups or a group's users.
+> More information: <https://manned.org/lid.8>.
```

Comment:
```suggestion
> More information: <https://manned.org/lid>.
```

Reply:
There exists `lid.3` which is completely unrelated. I think we should keep the section number.